web/modules/custom/joinup_sparql/joinup_sparql.post_update.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for Joinup SPARQL.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Update the Licence legal type vocabulary.
+ */
+function joinup_sparql_post_update_0107301(array &$sandbox): void {
```

Comment:
In Joinup, we are deploying using the `drush deploy` command (ref. https://www.drush.org/latest/deploycommand/).
This is adding the following sequence:
* Run hook_update_N functions.
* Run hook_post_update_NAME functions.
* Import configuration.
* Run hook_deploy_N functions.

Since post_updates run before configuration import, we generally prefer to run simple updates that use the API _after_ we run the configuration import. 
Please, move this function to the `joinup_core.deploy.php`. The numbering and sequence retain the mechanism of the `01_073_00` (without the underscores) that you wrote above. Also, the first update function, weather update, post update or deploy hook, starts with the 00, not the 01 from above.

Reply:
Please, note that, the `drush update` does not run the deploy hooks so it is not a Drupal Core official function. The `drush deploy` runs the full sequence.